src/ContentHubClient.php

```diff
@@ -1032,6 +1032,16 @@ protected function getExceptionMessage($method, array $args, \Exception $excepti
                       sprintf('Error trying to make a search query to Content Hub. Are your credentials inserted correctly? (Error Code = %d: %s)',
                         $response->getStatusCode(), $response->getReasonPhrase()));
 
+                case 'addEntitiesToInterestList':
+                    return $this->getErrorResponse($response->getStatusCode(),
+                      sprintf('Error trying to add entities to the interest list (Error Code = %d: %s)',
+                        $response->getStatusCode(), $response->getReasonPhrase()));
+
+                case 'deleteInterest':
+                    return $this->getErrorResponse($response->getStatusCode(),
+                      sprintf('Error trying to remove entity from the interest list (Error Code = %d: %s)',
+                        $response->getStatusCode(), $response->getReasonPhrase()));
+
```

Comment:
I think it's worth doing a small refactor here (if possible). I can see a repetitive pattern which could be abstracted out to make the code DRYer. AFAICS, each case is returning the exact same thing except for the message. I think we can benefit from the following pattern to reduce clutter:

``` php
switch ($decision) {
   case 'pattern-1':
      $message = 'some message goes here';
      break;
   case 'pattern-2':
      $message = 'some other message';
      break;
    ....
}

return $this->getErrorResponse(
   $response->getStatusCode(),
   $message,
   $response->getStatusCode(), $response->getReasonPhrase())
);
```

Reply:
@looking-4-justice I would say that this could be done in the scope of https://backlog.acquia.com/browse/LCH-2722, not the current one.

Reply:
Agreed!

Reply:
@looking-4-justice Actually, I've made a refactoring because of @saxumVermes mentioned `RequestException`. The solution was looking so horrible (too many duplications), so I decided to refactor this right now.